src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -17,7 +17,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
 {
-    internal partial class SymbolCompletionProvider : AbstractRecommendationServiceBasedCompletionProvider
+    internal partial class SymbolCompletionProvider : AbstractRecommendationServiceBasedCompletionProvider, IFeaturesCustomCommitCompletionProvider
```

Comment:
why?  SymbolCompletion is the most bog standard completion provider we have.  Why does it need CustomCommit capabilities?

Reply:
IFeaturesCustomCommitCompletionProvider provides "standard" commit capabilities. Some providers were excluded from the option to make this commit. I'm going to remove IFeaturesCustomCommitCompletionProvider and all provider will follow the approach. Thank you!

---
In reply to: [222523942](https://github.com/dotnet/roslyn/pull/29016#discussion_r222523942) [](ancestors = 222523942)

Reply:
Sounds good.  Thanks!